symfony/mercure-bundle/0.1/manifest.json

```diff
@@ -10,8 +10,25 @@
         "env(MERCURE_JWT_SECRET)": ""
     },
     "env": {
-        "MERCURE_PUBLISH_URL": "https://demo.mercure.rocks/hub",
-        "MERCURE_JWT_SECRET": "#see https://mercure.rocks"
+        "#1": "See https://symfony.com/doc/current/mercure.html#configuration",
+        "MERCURE_PUBLISH_URL": "http://mercure/hub",
+        "MERCURE_JWT_SECRET": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJtZXJjdXJlIjp7InB1Ymxpc2giOltdfX0.Oo0yg7y4yMa1vr_bziltxuTCqb8JVHKxp-f_FwwOim0"
```

Comment:
I don't know why this was never mentioned, but apparently the secret used here is:

`!ChangeMe!`

Reply:
It's mentioned here: https://github.com/symfony/recipes/pull/583/files#diff-d19ccce8daab2b00dbfa4e827ebde212R24
But indeed it must be added as a comment in `.env`. Can you open a PR please?